### PR TITLE
Add `MCNC/4K2022` tournament article

### PR DESCRIPTION
**Pull request by xz1z1z1z1z**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 1]
```diff
@@ -0,0 +1,623 @@
...
+| **Miyes** | **4** | 1 | N3k0ha_5h12uku | [#1](https://osu.ppy.sh/community/matches/102151683) |
+| **\[GB\]Tyris** | **4** | 1 | Molli | [#1](https://osu.ppy.sh/community/matches/102152440) |
+| **Zyuuu** | **4** | 0 | YUCInoback | [#1](https://osu.ppy.sh/community/matches/102154297) |
+| **- Minato  Aqua-** | **4** | 0 | shade0214 | [#1](https://osu.ppy.sh/community/matches/102155252) |
+| **chana** | **4** | 2 | MkoTenshi | [#1](https://osu.ppy.sh/community/matches/102155260) |
+| **YuLiangSSS** | **4** | 0 | katiexie | [#1](https://osu.ppy.sh/community/matches/102156436) |
+| **- Xiaoluoli -** | **4** | 1 | DJ Sharpnel | [#1](https://osu.ppy.sh/community/matches/102156358) |
+| **-Chen-** | **4** | 3 | Alptraum | [#1](https://osu.ppy.sh/community/matches/102157598) |
+
+### Qualifiers
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/4K2022/en.md:517]:
```suggestion
### Qualifiers
```
Inconsistent naming, if we're sticking with Line 20, then change it to Qualifier stage,
If we're sticking to Qualifiers, change the Line 20 one.

**Review by fajar13k — COMMENTED** [review]
Some piece

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 2]
```diff
@@ -0,0 +1,630 @@
...
+
+![MCNC 2022 logo](img/logo.png)
+
+The **osu!mania 4K Chinese National Cup 2022** (***MCNC 4K 2022***) was a country-based osu!mania tournament hosted by the \[Crz\]Team. It was the 4th instalment of the osu!mania 4K Chinese National Cup.
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-05-15/2022-06-12
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:19]:
```suggestion
| Registration phase | 2022-05-15/2022-06-12 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 3]
```diff
@@ -0,0 +1,630 @@
...
+| Qualifiers | 2022-06-26/2022-07-03
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:20]:
```suggestion
| Qualifiers | 2022-06-26/2022-07-03 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 4]
```diff
@@ -0,0 +1,630 @@
...
+| Round of 64 | 2022-07-09/2022-07-10
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:21]:
```suggestion
| Round of 64 | 2022-07-09/2022-07-10 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 5]
```diff
@@ -0,0 +1,630 @@
...
+| Round of 32 | 2022-07-16/2022-07-17
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:22]:
```suggestion
| Round of 32 | 2022-07-16/2022-07-17 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 6]
```diff
@@ -0,0 +1,630 @@
...
+| Round of 16 | 2022-07-23/2022-07-24
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:23]:
```suggestion
| Round of 16 | 2022-07-23/2022-07-24 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 7]
```diff
@@ -0,0 +1,630 @@
...
+| Quarterfinals | 2022-07-30/2022-07-31
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:24]:
```suggestion
| Quarterfinals | 2022-07-30/2022-07-31 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 8]
```diff
@@ -0,0 +1,630 @@
...
+| Semifinals | 2022-08-06/2022-08-07
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:25]:
```suggestion
| Semifinals | 2022-08-06/2022-08-07 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 9]
```diff
@@ -0,0 +1,630 @@
...
+| Finals | 2022-08-13/2022-08-14
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:26]:
```suggestion
| Finals | 2022-08-13/2022-08-14 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 10]
```diff
@@ -0,0 +1,630 @@
...
+| Grand Finals | 2022-08-20/2022-08-21
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:27]:
```suggestion
| Grand Finals | 2022-08-20/2022-08-21 |
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 11]
```diff
@@ -0,0 +1,630 @@
...
+1. There is no rank range for this tournament, players from all skill levels are free to participate.
+2. This tournament is for Chinese players only, including players from Mainland Chinese territories, Taiwan territories, Hong Kong territories, Macau territories. Please note that this tournament has nothing to do with politics and views about the territories of China. Only players with the flags of the territories listed above is eligible for playing.
+3. To be eligible for playing, one must:
+   - Have not been banned/majorly restricted in the MCNC.
+   - Pass a background check by osu!staff and the tournament staff.
+   - Include the answer "原来你是op " in the question "Do you have anything else to say?".
+4. To ensure that the game is played normally, every registered user will be manually checked by the Tournament Management and will give the list to the official osu! Staff to check.
+5. All successfully formed players will be published after the Registration Phase.
+6. Organisers, mappool selectors and referees must not participate as a player in the tournament.
+
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:572]:
```suggestion

### Qualifier instructions

```

Comment (RushFTK) [reply wiki/Tournaments/MCNC/4K2022/en.md:572]: 这些可以直接修正后面的github报错

**Review by RushFTK — CHANGES_REQUESTED** [review]
Suggestion for fix CI issues

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 12]
```diff
@@ -0,0 +1,632 @@
...
+   - Finals: 18 beatmaps
+   - Grand Final: 18 beatmaps
+4. Each mappool has one tiebreaker, except for the Qualifiers.
+5. Possible mod choices for all maps are Hidden, FadeIn, Flashlight, and Mirror.
+6. The tiebreaker will be played under FreeMod conditions.
+
+### Scheduling instructions
+1. Each stage will be held on a single weekend. It could be held on weekdays if each players required, but not recommend.
+2. The schedule is decided by the Tournament Management. The daily schedule will be released every Tuesday before.
+3. If player need to make a temporary change after decide the game time. Rescheduling can only be considered with the consent of both players, and the scheduled time must be every Thursday before.
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:632]:
```suggestion
3. If player need to make a temporary change after decide the game time. Rescheduling can only be considered with the consent of both players, and the scheduled time must be every Thursday before.

```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 13]
```diff
@@ -0,0 +1,632 @@
...
+2. Each player can ban one beatmap to be selected from the pool. These beatmaps can not be picked by any player in the entire match.
+3. Beatmap selection will alternate between each player selecting a beatmap in the mappool.
+4. Each player must use !roll once in #multiplayer.
+   - The winner of the !roll starts picking the first beatmap of the match.
+   - The loser of the !roll starts banning one beatmap, followed by the winner of the !roll to ban a beatmap.
+5. Players will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a player takes more time than alloted, for the second occurrence: a random map will be chosen from the mappool using !roll X, where X is the number of maps that were neither picked nor banned, excluding the Tiebreaker. Repeat offenders may receive further sanctions from the Tournament Management.
+6. Players will be given at most 3-minute pause for solving unexpected situations.
+7. Results of the Qualifiers Stage will be published via a Statistics sheet.
+
+### Mappool instructions
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:613]:
```suggestion
### Mappool instructions

```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 14]
```diff
@@ -0,0 +1,632 @@
...
+   - Round of 16: 16 beatmaps
+   - Quarterfinals: 16 beatmaps
+   - Semifinals : 18 beatmaps
+   - Finals: 18 beatmaps
+   - Grand Final: 18 beatmaps
+4. Each mappool has one tiebreaker, except for the Qualifiers.
+5. Possible mod choices for all maps are Hidden, FadeIn, Flashlight, and Mirror.
+6. The tiebreaker will be played under FreeMod conditions.
+
+### Scheduling instructions
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:629]:
```suggestion
### Scheduling instructions

```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 15]
```diff
@@ -0,0 +1,632 @@
...
+  7. [7\_7 - Mousou Chikan Express (frolica) \[Persecution Complex \[17\]\]](https://osu.ppy.sh/beatmapsets/1756404#mania/3594321)
+- LN
+  1. [B-ko (Cv:Touyama Nao) - Nisemono Chuuihou (juankristal) \[timing hell?\]](https://osu.ppy.sh/beatmapsets/1759390#mania/3600837)
+  2. [katagiri - Sendan Life (katagiri Bootleg) (\[Crz\]Alleyne) \[Raveille's NostaLNgia 1.1x (286bpm)\]](https://osu.ppy.sh/beatmapsets/1827506#mania/3750601)
+  3. [After the Rain - Anti-Clockwise (eZmmR) \[True Ending\]](https://osu.ppy.sh/beatmapsets/1802781#mania/3696755)
+  4. [Camellia – Enantiomorphs (Hylotl)
+ \[Diastereomers \[1.2x Rate\]\]](https://osu.ppy.sh/beatmapsets/1742086#mania/3561401)
+- Hybrid
+  1. [Camellia - +ERABY+E CONNEC+10N (HowToPlayLN) \[504 ServerError: Gateway Timeout\]](https://osu.ppy.sh/beatmapsets/1338625#mania/2772922)
+  2. [Camellia - Dans la mer de son (Toaph Daddy) \[Diabolos x1.1\]](https://osu.ppy.sh/beatmapsets/1443348#mania/2968916) https://osu.ppy.sh/beatmapsets/1338625#mania/2772922)
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:164]:
```suggestion
  2. [Camellia - Dans la mer de son (Toaph Daddy) \[Diabolos x1.1\]](https://osu.ppy.sh/beatmapsets/1443348#mania/2968916)
```

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 16]
```diff
@@ -0,0 +1,632 @@
...
+
+### Round of 16
+
+**[Download the mappack here! (166 MB)](https://drive.google.com/file/d/1YIkSmbmRVRLAffl8X7I03Rf_n5ageXOV/view?usp=sharing)**
+
+- Rice
+  1. [E-Type - This ls The Way (Jole) \[short way\]](https://osu.ppy.sh/beatmapsets/1435426#mania/3000972)
+  2. [DJ SHARPNEL- STRANGEPROGRAM (Shoegazer) \[Malfunction\]](https://osu.ppy.sh/beatmapsets/1393835#mania/2888756)
+  3. [DJ SHARPNEL- Kaze no Naka no Subaru"Hyougenshatachi~ (Jole) \[subarushii\]](https://osu.ppy.sh/beatmapsets/1628745#mania/3325116)
+  4. [ F-777 - Ultra Violet (chxu) \[Hyperbolic x1.05\]](https://osu.ppy.sh/beatmapsets/1687224#mania/3448235)
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:267]:
```suggestion
  4. [F-777 - Ultra Violet (chxu) \[Hyperbolic x1.05\]](https://osu.ppy.sh/beatmapsets/1687224#mania/3448235)
```
我实在看不出来这里为啥会报错，只能蒙一个了

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 17]
```diff
@@ -0,0 +1,632 @@
...
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **\[GB\]gluefly** | **0** | -1 | Hunxin | *Win by Default* |
+| **Wings_AnChen** | **6** | 5 | \[JbL\]Hx_032 | [#1](https://osu.ppy.sh/community/matches/102466607) |
+| **\[Crz\]lf1f1f** | **6** | 0 | YUCInoback | [#1](https://osu.ppy.sh/community/matches/102467659) |
+| **Zyuuu** | **6** | 0 | Etanity Larva | [#1](https://osu.ppy.sh/community/matches/102467716) |
+| **- Xiaoluoli -** | **6** | 1 | N3k0ha_5h12uku | [#1](https://osu.ppy.sh/community/matches/102468766) |
+| **rurudo1o1o** | **6** | 2 | MkoTenshi | [#1](https://osu.ppy.sh/community/matches/102468763) |
+| **Kuiiiiteeee** | **6** | 1 | \[Crz\]Riko | [#1](https://osu.ppy.sh/community/matches/102470373) |
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:419]:
```suggestion
| **Kuiiiiteeee** | **6** | 1 | \[Crz\]Riko | [#1](https://osu.ppy.sh/community/matches/102470373) |
```

**Review by RushFTK — CHANGES_REQUESTED** [review]
Try fix github CIs

File: wiki/Tournaments/MCNC/4K2022/en.md [hunk 18]
```diff
@@ -0,0 +1,635 @@
...
+4. Each mappool has one tiebreaker, except for the Qualifiers.
+5. Possible mod choices for all maps are Hidden, FadeIn, Flashlight, and Mirror.
+6. The tiebreaker will be played under FreeMod conditions.
+
+### Scheduling instructions
+
+1. Each stage will be held on a single weekend. It could be held on weekdays if each players required, but not recommend.
+2. The schedule is decided by the Tournament Management. The daily schedule will be released every Tuesday before.
+3. If player need to make a temporary change after decide the game time. Rescheduling can only be considered with the consent of both players, and the scheduled time must be every Thursday before.
+
```
Comment (RushFTK) [inline wiki/Tournaments/MCNC/4K2022/en.md:635]:
```suggestion
3. If player need to make a temporary change after decide the game time. Rescheduling can only be considered with the consent of both players, and the scheduled time must be every Thursday before.
```
我吐了，我刚才看了看其他的文件确认了一下，现在md文件最后一行又不用留空白行了，还得把635行删掉。。。

Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/4K2022/en.md]: Done

**Comment by RushFTK** [comment]
注意现在的PR报错， 文件本身应该没问题了
```

Error: You have edited some original articles (en.md), but did not outdate their translations:
* wiki/Tournaments/de.md
* wiki/Tournaments/fr.md

If your changes DON'T NEED to be added to the translations, add SKIP_OUTDATED_CHECK anywhere in the description of your pull request.
Otherwise, add the following to each article's front matter (https://osu.ppy.sh/wiki/en/Article_styling_criteria/Formatting#front-matter):

---
outdated_since: edb1fdc556e783ed74a[27](https://github.com/ppy/osu-wiki/actions/runs/3512155485/jobs/5883577228#step:13:28)8b8f5c6ca44161ba995
outdated_translation: true
---
Error: Process completed with exit code 1.
```

**Review by TPGPL — APPROVED** [review]

**Pull request merged**